packages/toolbox-core/src/toolbox_core/client.ts

```diff
@@ -159,40 +183,59 @@ class ToolboxClient {
    * tool remotely.
    *
    * @param {string} name - The unique name or identifier of the tool to load.
-   * @param {BoundParams} [boundParams] - Optional parameters to pre-bind to the tool.
+   * @param {AuthTokenGetters | null} [authTokenGetters] - Optional map of auth service names to token getters. Can be null.
```

Comment:
```suggestion
   * @param {AuthTokenGetters | null} [authTokenGetters] - Optional map of auth service names to token getters.
```
nit: This is implied through the word `"Optional"`, and is also not present for the parameter below.